### PR TITLE
Add Jawi Malay to posting languages

### PR DESCRIPTION
**Pull request by gunchleoc**
Requested via Mastodon: https://mastodon.social/@msiapublictransport/111818657390526946

I have chosen "Jawi Malay" as the English name, because I expect that this will be a more likely search term that users would try.

CLDR reference: https://www.unicode.org/cldr/charts/44/summary/ms_Arab.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mastodon/mastodon",
   "license": "AGPL-3.0-or-later",
-  "packageManager": "yarn@4.0.2",
+  "packageManager": "yarn@4.1.0",
```
Comment (renchap) [inline package.json:4]: This change does not belong to this PR, please rebase without it

**Review by renchap — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gunchleoc) [reply package.json]: Oops, done.

**Review by renchap — APPROVED** [review]
